applications/StructuralMechanicsApplication/custom_strategies/custom_strategies/prebuckling_strategy.hpp

```diff
@@ -0,0 +1,980 @@
+
+        ModelPart &rModelPart = BaseType::GetModelPart();
+        ProcessInfo& rProcessInfo = rModelPart.GetProcessInfo();
+
+        const int rank = rModelPart.GetCommunicator().MyPID();
+
+        SchemePointerType &pScheme = this->pGetScheme();
+        BuilderAndSolverPointerType &pBuilderAndSolver = this->pGetBuilderAndSolver();
+        //SparseMatrixType &rMassMatrix = this->GetMassMatrix();
+        SparseMatrixType &rStiffnessMatrix = this->GetStiffnessMatrix();
+        SparseMatrixType &rStiffnessMatrixPrevious = this->GetStiffnessMatrixPrevious();
+        SparseVectorType &rDx = this->GetDx();
+        SparseVectorType &rRHS = this->GetRHS();
+
+        typename ConvergenceCriteriaType::Pointer pConvergenceCriteria = mpConvergenceCriteria;
+
+        //initializing the parameters of the Newton-Raphson cycle
+        unsigned int iteration_number = 1;
+        rModelPart.GetProcessInfo()[NL_ITERATION_NUMBER] = iteration_number;
+        bool is_converged = false;
+        
+        // store inital load condition values
+        if( mLoadStepIteration == 0)
+        {
+            StoreInitialLoadConditions();
+        }
+        // update load conditions
+        // implemented so far are point and surface loads
+        // don't change forces in the first load step
+        if( mLoadStepIteration > 0)
+        {
+            UpdateLoadConditions();
+        }
+
+        // update loadfactor increment
+        double deltaLoadMultiplier = 0.0;
+        if( mLoadStepIteration == 1) // inital step
+        {
+            deltaLoadMultiplier = mInitialStep*(lambda + lambdaPrev);
+        }
+        else if( mLoadStepIteration % 2 == 1 ) //small step
+        {
+            deltaLoadMultiplier = mSmallStep*(lambdaPrev );
+        }
+
+        BuiltinTimer system_solve_time;
+        // initialize nonlinear iteration
+        this->pGetScheme()->InitializeNonLinIteration( rModelPart,rStiffnessMatrix, rDx, rRHS );
+        pConvergenceCriteria->InitializeNonLinearIteration(rModelPart, pBuilderAndSolver->GetDofSet(), rStiffnessMatrix, rDx, rRHS);
+        is_converged = mpConvergenceCriteria->PreCriteria(rModelPart, pBuilderAndSolver->GetDofSet(), rStiffnessMatrix, rDx, rRHS);
+
+        TSparseSpace::SetToZero(rStiffnessMatrix);
+        TSparseSpace::SetToZero(rRHS);
+        TSparseSpace::SetToZero(rDx);
+        // build system and solve system
+        pBuilderAndSolver->BuildAndSolve(pScheme, rModelPart, rStiffnessMatrix,rDx, rRHS);
+        // Update internal variables
+        pScheme->Update(rModelPart, pBuilderAndSolver->GetDofSet(), rStiffnessMatrix, rDx, rRHS);
+        BaseType::MoveMesh();
+        // finalize nonlinear solution step
+        pScheme->FinalizeNonLinIteration( rModelPart,rStiffnessMatrix, rDx, rRHS );
+        pConvergenceCriteria->FinalizeNonLinearIteration(rModelPart, pBuilderAndSolver->GetDofSet(), rStiffnessMatrix, rDx, rRHS);
+                   
+        if (is_converged)
+        {
```

Comment:
In the style guide the { is in the declaration

Reply:
Sry, I didn't see it in the style guide. That means in every scope (class, function, if statement etc. ) the "{" needs to be one line up in the declaration?